database/seeds/eggs/voice-servers/egg-teamspeak3-server.json

```diff
@@ -3,34 +3,34 @@
+            "container": "ubuntu:18.04",
+            "entrypoint": "bash"
         }
     },
     "variables": [
         {
-            "name": "Server Version",
-            "description": "The version of Teamspeak 3 to use when running the server.",
-            "env_variable": "TS_VERSION",
-            "default_value": "3.5.0",
+            "name": "Update",
```

Comment:
I commented on defaulting to latest but letting the user choose and you did this instead.